.travis.yml

```diff
@@ -25,7 +25,7 @@ stages:
   - Run unattended openHABian installation
 
 install:
-# prepare configuration for tests, select debug level here:
+  # prepare configuration for tests, select debug level here:
```

Comment:
General comment to travis.yml:
Please add an overview of intended tests, e.g.
rpi0/1   RPi 0, RPi1           native ARMvxxxx        32bit OS   32 bit Azul Java
rpi3      Rpi3, RPi4             native ARMyyy           32bit OS  32 bit Azul Java
....

I am getting confused....

Reply:
me, too :)
added comment lines for all test cases